pom.xml

```diff
@@ -734,5 +735,48 @@
                 </repository>
             </repositories>
         </profile>
+        <profile>
+            <id>run-proguard</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
```

Comment:
Shouldn't this be true?

Reply:
Well, if you leave it off by default, you'll need to change the workflow for the extensions to enable the profile. If it's on by default, you'd have to change the workflow of the parent pom to disable the profile. Pick your poison. Either way, workflow changes need to be made somewhere.

I was thinking on-by-default because it would follow the pattern established in the pro repo.

Reply:
Okay, that makes sense.